adfrontend/fdnd.nl
# Performance fixes

# What does this PR do

- Add webp images
- Changed HTML img tags to picture tags
- Add local Open Sans variable fonts
- Add @font-face normal and italic to CSS 
- Add preload of most used font to html to prevent layout shift
- Removed Google fonts links from HTML

# How to test

1. You can check the changes in the code and see if anything stands out.
2. You can run this version local and put the local and live side by side to spot any differences.

# Related issue

#16 #15 

# Checks

## Code quality

- [x] This branch is up to date with `source`.
- [x] Code is double checked for debugging stuff, and self-reviewed.
- [x] Comments are added where necessary to review the PR.

## Extra's

- [x] Reviewer is appointed.
